Missing types for the new `manager-helpers`

Missing types for the new `manager-helpers`.

```
.storybook/manager.ts:4:8 - error TS7016: Could not find a declaration file for module 'storybook-addon-tag-badges/manager-helpers'. /node_modules/.pnpm/storybook-addon-tag-badges@3.0.1_storybook@10.0.0_@testing-library+dom@10.4.1_prettier@_5f72032ef56bef11c7754fc52e97a520/node_modules/storybook-addon-tag-badges/dist/manager-helpers.js' implicitly has an 'any' type.

  Try `npm i --save-dev @types/storybook-addon-tag-badges` if it exists or add a new declaration (.d.ts) file containing `declare module 'storybook-addon-tag-badges/manager-helpers';`

4 } from "storybook-addon-tag-badges/manager-helpers";
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in .storybook/manager.ts:4
```

--- a/tsup.config.ts
+++ b/tsup.config.ts
@@ -26,13 +26,13 @@ export default defineConfig(async () => {
 
   // manager entries are entries meant to be loaded into the manager UI
   // they'll have manager-specific packages externalized and they won't be usable in node
-  // they won't have types generated for them as they're usually loaded automatically by Storybook
   if (managerEntries.length) {
     configs.push({
       ...commonConfig,
       entry: managerEntries,
       platform: 'browser',
       target: 'esnext',
+      dts: true,
     })
   }
 

